src/Sql/Sql/Az.Sql.psd1

```diff
@@ -271,7 +272,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Support Database Data Classification.'
+        ReleaseNotes = '* Add Virtual Cluster cmdlets'
```

Comment:
Please revert this change, we will update during versioning.

Reply:
Done, thanks.